app/models/chargeback_rate_detail.rb

```diff
@@ -2,7 +2,8 @@ class ChargebackRateDetail < ActiveRecord::Base
   belongs_to :chargeback_rate
   belongs_to :detail_measure, :class_name => "ChargebackRateDetailMeasure", :foreign_key => :chargeback_rate_detail_measure_id
 
-  validates_numericality_of :rate
+  validates :rate, :numericality => true
```

Comment:
Seems like the test on the API side will indirectly test the validations. I think that as long as we are testing for the expected behavior through an invalid request we should be good.

With that said, it wouldn't hurt to extend that test to test make sure the expected validation failure message is returned in the response because that's what's important for an API user.